integration_tests/features/contexts.feature

```diff
@@ -0,0 +1,16 @@
+
+# Aruba reference
+# https://gist.github.com/bdunn313/4199906
+
+Feature: Context integration tests
+
+  @mocked_home_directory
+  Scenario: when listing contexts without a token
+    When I run `circleci context list github foo --skip-update-check --token ""`
```

Comment:
@aengelberg wdyt of using a cucumber test for this?

Reply:
Didn't know about cucumber till now, but I don't see why not :+1: